website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -187,6 +187,12 @@ The `Dropdown::ListItem::Interactive` component, yielded as contextual component
 It internally uses the [`Hds::Interactive`](/utilities/interactive) utility component. For more details about this component API, please refer to [its documentation page](/utilities/interactive?tab=code#component-api).
 
 <Doc::ComponentApi as |C|>
+  <C.Property @name="yield">
+    Elements passed as children are yielded as inner content of `"interactive"` block.
```

Comment:
Maybe opinionated here (others feel free to chime in), but I don't think the additional quotes inside of back ticks are needed here when a property/variant is being referred to.
```suggestion
    Elements passed as children are yielded as inner content of `interactive` block.
```

Reply:
I agree with that